content/guides/process.md

```diff
@@ -31,22 +88,20 @@ and adding the following line to the `[Main]` section
     
 After configuration is complete, restart the Agent.
 
-## Docker container
+**Docker container**
 
-**Update to the Datadog Agent image version 5.16.0 or above**
+Update to the Datadog Agent image version 5.16.0 or above:
```

Comment:
maybe for the next pass but we could call out 5.17 which will:

* have docker collection on by default
* won't need the HOST_PROC/HOST_SYS environment if you're running inside a container

5.17 should go on Wednesday